msg/GimbalControls.msg

```diff
@@ -4,4 +4,4 @@ uint8 INDEX_PITCH = 1
 uint8 INDEX_YAW = 2
 
 uint64 timestamp_sample	    # the timestamp the data this control response is based on was sampled
-float32[3] control
+float32[3] control	# Control input in [-1, 1] maps to min/max deflection.
```

Comment:
You could use the same definition as in ActuatorServo.msg:
```suggestion
float32[3] control	# [-] [@range -1, 1] Normalized output. 1 means maximum positive position. -1 maximum negative position. NaN maps to disarmed.
```

Reply:
Actually, given that in output_rc.cpp you do the mapping like that:
```
	if (value >= FLT_EPSILON) {
		return math::interpolate(value, 0.f, max_value, 0.f, 1.f);

	} else {
		return math::interpolate(value, min_value, 0.f, -1.f, 0.f);
	}
```
You need to specify in the message definition that controls=0 means "no deflection".